2-collectors/scala-stream-collector/project/Dependencies.scala

```diff
@@ -33,11 +33,15 @@ object Dependencies {
     val awsSdk           = "1.6.10"
     val yodaTime         = "2.1"
     val yodaConvert      = "1.2"
+    val kafka            = "0.10.1.0"
     // Scala
+    val snowplowCommonEnrich = "0.22.0"
```

Comment:
I was unsure how to version this so I made a major version increment but kept the minor version so we could know which minor version we forked from. I'd love a better suggestion as to how this should be versioned